pacai/bin/eightpuzzle.py

```diff
@@ -252,18 +252,17 @@ def createRandomEightPuzzle(moves=100):
 
 if __name__ == '__main__':
   puzzle = createRandomEightPuzzle(25)
-  print('A random puzzle:')
-  print(puzzle)
+  logging.basicConfig(format='%(levelname)s - %(asctime)s - %(message)s', level=logging.INFO)
```

Comment:
This should be a print, not a log.
Printing the puzzle like this is part of the core UI in eightpuzzles.

In fact, I think all the logs in this file should go back to prints.
This bin package will probably have the most prints, where as the other packages will most likely be exclusively logs.